src/fetch.rs

```diff
@@ -266,9 +266,21 @@ fn fuzzy_query_registry_index(
     registry_path: impl AsRef<Path>,
 ) -> Result<Vec<CrateVersion>> {
     let crate_name = crate_name.into();
+    let remotes = PathBuf::from("refs/remotes/origin/");
+    let checkout_dir = registry_path.as_ref().join(".git").join(&remotes);
+    let checkout = checkout_dir
+        .read_dir()?
+        .next() // Is there always only one branch? (expecting either master og HEAD)
```

Comment:
I think it would be safer to try only known names `master` and if it fails, then `HEAD`

Reply:
I think this is mostly ok. I wanted to know if there are any reference on how cargo deals according to this. As I think most users will never toutch this location I think it should be fine. If we want to check for names i would rather collect them into a vec and then first check `master` then `HEAD` then first element. Not sure about that order either.